### ISSUE TITLE
Not working properly inside SwipeRefreshLayout 

### ISSUE THREAD

**Issue by malikmotani** [amarjain07/StickyScrollView#13]
While pull to refresh with `StickyScrollView` footer is getting blink at the bottom. I have tried `StickyScrollView extends NestedScrollView`, still getting the same results.

**Comment by malikmotani** [comment]
I have resolved it in this way

```
 boolean isFooterAdded = false;
 @Override
 protected void onLayout(boolean changed, int l, int t, int r, int b) {
        super.onLayout(changed, l, t, r, b);
        if (stickyFooterView != null && !changed && !isFooterAdded) {
            stickyFooterView.getLocationInWindow(updatedFooterLocation);
            mStickyScrollPresenter.recomputeFooterLocation(getRelativeTop(stickyFooterView),
                updatedFooterLocation[1]);
            isFooterAdded = true;
        }
 }
```

**Comment by amarjain07** [comment]
Fixed in v1.0.3
